# RadarPointArray.msg
# 定义雷达点数组消息类型

blue_teeth_pkg/RadarPoint[] points